<launch>
        # Node
        <node name="drivenet_node_a" pkg="drivenet" type="drivenet_node_a"  output="screen">
                <param name="car_id" type="int" value="1" />
                <param name="standard_fps" type="bool" value="0" />
                <param name="display" type="bool" value="0" />
                <param name="input_resize" type="bool" value="1" />
                <param name="imgResult_publish" type="bool" value="1" />
                <param name="dist_esti_mode" type="int" value="0" />
        </node>

</launch>

